bin/einhorn

```diff
@@ -314,6 +314,10 @@ if true # $0 == __FILE__
       Einhorn::State.config[:max_unacked] = Integer(n)
     end
 
+    opts.on('--max-additional-while-upgrading=N', 'Maximum number of additional workers that can be running during an upgrade.') do |n|
```

Comment:
Personally, I like descriptive names, but I think this one may be just a little too unwieldy for a command line argument. What do you think about `--max-upgrade-additional` or something like it?

(I tried to find some naming precedent from us to reuse from similar projects like Unicorn, but couldn't.)

Reply:
Sure, I have no preference on the names. I changed it to `--max-upgrade-additional`